src/Symfony/Bundle/FrameworkBundle/Resources/config/cache_pools.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+
+        <service id="cache.adapter.default" alias="cache.adapter.filesystem" />
+
+        <service id="cache.adapter.apcu" class="Symfony\Component\Cache\Adapter\ApcuAdapter" abstract="true">
+            <tag name="cache.pool" namespace-arg-index="0" default-lifetime-arg-index="1" />
+            <argument /> <!-- namespace -->
+            <argument /> <!-- default lifetime -->
```

Comment:
In other services `namespace` is second argument but here is first. Maybe better change constructor signature for better consistency?